common/src/main/java/com/wynntils/mc/mixin/LocalPlayerMixin.java

```diff
@@ -49,4 +50,23 @@ private void playSoundPre(SoundEvent sound, float volume, float pitch, CallbackI
             ci.cancel();
         }
     }
+
+    @Inject(
+            method = "Lnet/minecraft/client/player/LocalPlayer;sendPosition()V",
+            at = {
+                @At(
+                        value = "NEW",
+                        target = "(DDDFFZ)Lnet/minecraft/network/protocol/game/ServerboundMovePlayerPacket$PosRot;"),
+                @At(
+                        value = "NEW",
+                        target = "(DDDFF)Lnet/minecraft/network/protocol/game/ServerboundMovePlayerPacket$Pos;"),
```

Comment:
I am wondering how you got this signature. I checked and the constructor doesn't even have that many variables.

 ```suggestion
                        target = "(DDDZ)Lnet/minecraft/network/protocol/game/ServerboundMovePlayerPacket$Pos;"),
```

Can you confirm which one is correct?

Reply:
Probably copilot and I didn't notice since the first one gets tripped a lot anyway. DDDZ is correct.